distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFtp.java

```diff
@@ -264,4 +432,63 @@ private void showServerReply(FTPClient ftpClient) {
             }
         }
     }
-}
+
+    /**
+     * Sets the clientAuth configuration in catalog-ftp feature.
+     *
+     * @throws Exception
+     */
+    private void setClientAuthConfiguration(String clientAuth) throws Exception {
+        Configuration config =
+                getAdminConfig().getConfiguration("ddf.catalog.ftp.FtpServerStarter");
+        config.setBundleLocation("mvn:ddf.catalog/ftp/" + System.getProperty("ddf.version"));
+        Dictionary properties = new Hashtable<>();
+
+        properties.put("port", FTP_DEFAULT_PORT);
```

Comment:
I know you didn't hardcode this port number but we really need to change it to a DynamicPort https://github.com/codice/ddf/blob/4c6e861bf6689d82bc53271582c61ce55d58d26d/distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java#L125